.editorconfig

```diff
@@ -4,6 +4,6 @@ root = true
 charset = utf-8
 end_of_line = lf
 indent_size = tab
-indent_style = space
+indent_style = tab
```

Comment:
I had not noticed this but now that you mention it this is a pertinent fix. Thanks for catching it!